751: Correct video player size

bronzehedwick opened the pull request:
Add some sane defaults for `video` element sizes, so they can't blow out their container. Add some extra constraints for videos inside horizontal cards (browse, dh browse pages), so vertical videos don't get too long in a summary view.

Fixes #751

bronzehedwick committed:
751: Correct video player size

Add some sane defaults for `video` element sizes, so they can't blow out
their container. Add some extra constraints for videos inside horizontal
cards (browse, dh browse pages), so vertical videos don't get too long
in a summary view.

Fixes #751

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr771-zmd86mzqyxic9hl9jazis6evx4qjdpko.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67e5678a908d23476b16f6b6

michael-wynne-wsu commented:
@christa-kavanagh-wsu when you're available, can you test with that really tall skinny video? I don't have any weird proportion videos to test with.

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr771-zmd86mzqyxic9hl9jazis6evx4qjdpko.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67e5678a908d23476b16f6b6

christa-kavanagh-wsu commented:
List and grid work fine for me with both videos now. Thanks, @bronzehedwick! 
![image](https://github.com/user-attachments/assets/c6a72647-b521-4661-9756-fb2656606fca)
The only place things are still showing up a little funky is the tall video on the map. Not sure whether to address that here or elsewhere.
![image](https://github.com/user-attachments/assets/4337e527-0d29-4aeb-9a08-4e3ba555d6f7)

bronzehedwick committed:
751: Reduce leaflet preview video height

Also fix a bug where the map preview title does not always have white
space above it.

michael-wynne-wsu commented:
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr771-zmd86mzqyxic9hl9jazis6evx4qjdpko.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/67e5678a908d23476b16f6b6

bronzehedwick commented:
Ha, we were commenting at the same time @michael-wynne-wsu! I think the fix I pushed should address what you're speaking about; I added a max height to the videos inside the leaflet previews (along with a bugfix for the title white space).

christa-kavanagh-wsu commented:
Thanks @bronzehedwick, it looks like it's fixed!
![image](https://github.com/user-attachments/assets/ab5360c9-806a-4aab-8355-8a0bc6e49aa5)
